authors/apps/articles/tests/test_article_api_endpoints.py

```diff
@@ -256,3 +257,8 @@ def test_user_can_the_number_of_people_that_disliked_an_article(self):
         """a user can get an article's dislike count"""
         response = self.client.get(self.article_url(self.article.slug))
         self.assertEqual(response.data['dislike_count'], 0)  # we haven't disliked any article yet
+
+    def test_get_tags_when_none_exists(self):
```

Comment:
@FahdJamy I think you meant to put this test in the upper class. This class includes tests for the like, `dislike` `functionality` only.